client/src/backoffice/TransactionsPage.tsx

```diff
@@ -88,14 +92,16 @@ const TransactionsPage: FC = () => {
   };
 
   const handleFilterToggle = (status: string) => {
-    let newStatusList = [...statusList];
-    let index = newStatusList.indexOf(status);
-    if (index > -1) {
-      newStatusList.splice(index, 1);
+    const _statusList = [...statusList];
+    const isStatusInStatusList = _statusList.indexOf(status) > -1;
+    const indexOfStatus = _statusList.indexOf(status);
+
+    if (isStatusInStatusList) {
+      _statusList.splice(indexOfStatus, 1);
```

Comment:
Should the `indexOfStatus` live inside the conditional ? Since that's were you are going to need it. Also you can use `.includes()` method instead of `indexOf`

```typescript
const _statusList = [...statusList];
const isStatusInStatusList = _statusList.includes(status);

if (isStatusInStatusList) {
  const indexOfStatus = _statusList.indexOf(status);
  _statusList.splice(indexOfStatus, 1);
}
```